devel/xxdiff/Portfile

```diff
@@ -1,62 +1,100 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
+# -*- coding: utf-8; mode: tcl; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4; truncate-lines: t -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           qt5 1.0
 
 name                xxdiff
-version             3.2
-revision            1
+# the current xxdiff version is 5.0b1, which we approach numerically with 4.9.90
+version             4.9.90
 categories          devel
-platforms           darwin
 license             GPL-2+
-maintainers         nomaintainer
-
+platforms           darwin
+maintainers         gmail.com:rjvbertin openmaintainer
 description         xxdiff is a graphical merging tool
 long_description    xxdiff is a graphical browser for viewing the \
-                    differences between two or three files, or between \
-                    two directories, and can be used to produce a \
-                    merged version.
+                    differences between two or three files, or between two \
+                    directories, and can be used to produce a merged version. \
+                    It is inspired by the venerable SGI xdiff utility
+
 homepage            http://furius.ca/xxdiff/
 
-depends_lib         port:bison \
-                    port:flex \
-                    port:qt3 \
-                    port:libGLU
 
-# xxdiff links with libGLU if it is installed; so make sure it is
-# installed already.  See also: https://trac.macports.org/ticket/47693
+depends_lib         port:bison \
```

Comment:
`depends_lib` should probably be `depends_lib-append` when you want to keep the qt5 dependency from the port group.